### PR TITLE
[WIP][No-3][Training]init_layout_admin[Review]

### PR DESCRIPTION
**Pull request by phantuan1803**
- Task name: Init layout admin
- Document: https://laravel.com/docs/6.x/blade 
- Content need review: layout admin (manager user)
- Checked with phpcs PSR-2
- Command to create layout: No

### CODE REVIEW

**Review by nguyentienthinh — APPROVED** [review]
LGTM

**Review by loitq — COMMENTED** [review]
LGTM
Please send to next reviewer.

**Comment by nguyentienthinh** [comment]
LGTM

**Comment by HoangBM** [comment]
Please help me update pull request description

- Task name
- Document
- Content need review
- Check with PHPCS, MD

QA:
- Could you tell me the list of commands to create this layout?

File: resources/views/admin/layout/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    </script>
+
+    <!-- SB Admin Scripts - Include with every page -->
+    <script src="/asset_admin/js/sb-admin.js">
+    </script>
+
+    <!-- Page-Level Demo Scripts - Tables - Use for reference -->
+    <script>
+    $(document).ready(function() {
+        $('#dataTables-example').dataTable();
```
Comment (HoangBM) [inline resources/views/admin/layout/index.blade.php]: Where are you using this id?

File: resources/views/admin/user/list.blade.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                                        <th>Email</th>
+                                        <th>Password</th>
+                                        <th></th>
+                                        <th></th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    <tr class="odd gradeX">
+                                        <td>1</td>
+                                        <td>Phan Quoc Tuan</td>
```
Comment (HoangBM) [inline resources/views/admin/user/list.blade.php]: Why are you using mock data in here?

**Review by HoangBM — CHANGES_REQUESTED** [review]
Please check.

Maybe miss auth for the admin role.

File: resources/views/admin/user/list.blade.php (hunk 2 above)
Comment (phantuan1803) [reply resources/views/admin/user/list.blade.php]: This is just an interface initialization, this is just fake data of the user management screen, not to get the database

File: resources/views/admin/layout/index.blade.php (hunk 1 above)
Comment (phantuan1803) [reply resources/views/admin/layout/index.blade.php]: Yes, I will delete it, it not used

File: resources/views/admin/user/list.blade.php (hunk 2 above)
Comment (HoangBM) [reply resources/views/admin/user/list.blade.php]: No need using mock data

Comment (phantuan1803) [reply resources/views/admin/user/list.blade.php]: I fixed, no need mock data, only layout

**Pull request merged**